app/assets/javascripts/listings/ListingLotteryService.js.coffee

```diff
@@ -40,7 +40,7 @@ ListingLotteryService = ($http, ListingIdentityService, ModalService) ->
 
   Service.lotteryIsUpcoming = (listing) ->
     return false unless listing
-    !listing.Lottery_Results && !Service.lotteryDatePassed(listing)
+    !listing.Lottery_Results && !Service.lotteryDatePassed(listing) && !Service.lotteryComplete(listing)
```

Comment:
@akegan I forgot to ask about this one. Should I remove `lotteryDatePassed` to fulfill condition from ticket's acceptance criteria?

Reply:
no I think this is good as is